core/src/Platforms/iOS/EmbeddedWebview/AuthenticationAgentUIViewController.cs

```diff
@@ -25,28 +25,23 @@
 //
 //------------------------------------------------------------------------------
 
-using System;
-using System.Collections.Generic;
-using System.Globalization;
-using CoreFoundation;
-using CoreGraphics;
 using Foundation;
+using Microsoft.Identity.Core.Platforms.iOS.EmbeddedWebview;
+using System;
 using UIKit;
-using Microsoft.Identity.Core.Helpers;
+using WebKit;
+using static Microsoft.Identity.Core.Platforms.iOS.EmbeddedWebview.WKWebNavigationDelegate;
 
 namespace Microsoft.Identity.Core.UI.EmbeddedWebview
```

Comment:
>Microsoft.Identity.Core.UI.EmbeddedWebview [](start = 10, length = 42)

seems like the two namespaces for the WKWebNavigationDeletate and this class is not the same. Any reason why?

Reply:
fixed...thanks.